manim-latex/tools/chocolateyinstall.ps1

```diff
@@ -4,6 +4,6 @@ Start-ChocolateyProcessAsAdmin `
   -Statements "tlmgr update --self" `
   -Elevated
 Start-ChocolateyProcessAsAdmin `
-  -Statements "tlmgr install standalone everysel ctex frcursive preview doublestroke ms setspace rsfs relsize ragged2e fundus-calligra microtype wasysym physics dvisvgm jknapltx wasy cm-super babel-english gnu-freefont mathastext cbfonts-fd" `
+  -Statements "tlmgr install amsmath fontspec latex-bin tipa xcolor xetex xkeyval standalone everysel ctex frcursive preview doublestroke ms setspace rsfs relsize ragged2e fundus-calligra microtype wasysym physics dvisvgm jknapltx wasy cm-super babel-english gnu-freefont mathastext cbfonts-fd" `
```

Comment:
Pretty much all of the packages there should be installed already when installing tinytex. See https://github.com/yihui/tinytex/blob/main/tools/pkgs-custom.txt